Initium-Odp/src/com/universeprojects/miniup/server/controllers/SublocationsController.java

```diff
@@ -44,12 +44,21 @@ protected final String processRequest(HttpServletRequest request, HttpServletRes
 		ODPDBAccess db = ODPDBAccess.getInstance(request);
 		try{InitiumPageController.requireLoggedIn(db);}catch(NotLoggedInException e){return InitiumPageController.loginMessagePage;}
 		
+		
+	    CachedEntity character = db.getCurrentCharacter();
+	    
+	    if(CommonChecks.checkCharacterIsIncapacitated(character)) {
+	    	request.setAttribute("locationsViewable", false);
```

Comment:
Make sure attribute is set to true in the else case, otherwise sublocations will never be viewable.